LaTeX/LaTeX Log.sublime-syntax

```diff
@@ -6,44 +6,81 @@ scope: text.log.latex
 first_line_match: "This is (pdf|pdfe)?TeXk?, Version "
 
 contexts:
+  eol-pop:
+    - match: '$\n?'
+      pop: 1
+
   main:
+    - include: invalids
+    - include: functions
+    - include: types
+    - include: tags
+    - include: comments
+
+    - match: ".*FiXme:"
+      scope: meta.log.latex.fixme
+
+    - match: (?:Overfull|Underfull)
+      scope: keyword.control.hyphenation.latex
+      push:
+        - meta_scope: meta.log.latex.hyphenation
+
+        - match: \[\]\n
+          scope: keyword.control.hyphenation.latex
+          pop: 1
+
+        - match: '[0-9]+\-\-[0-9]+'
+          scope: variable.parameter.hyphenation.latex2
+
+    - include: strings
+
+  invalids:
     - match: ".*Warning:"
       scope: invalid.deprecated
+
     - match: '[^:]*:\d*:.*'
       scope: invalid.deprecated
+
     - match: .*Error|^!.*
       scope: invalid.illegal
```

Comment:
These are just log entry categories. `.*Info.*` from line 63 belongs to them as well as the `.*Fixme:` pattern does.

![grafik](https://user-images.githubusercontent.com/16542113/202715294-da91473c-4c22-4d31-985e-79c4788a7fc8.png)